README.md

```diff
@@ -160,6 +160,21 @@ No data should be lost since it is just re-generateable build files.
 [pythondevmode]: https://docs.python.org/3/library/devmode.html
 [pudb]: https://pypi.org/project/pudb/
 
+## Setting up a development environment
```

Comment:
This is still WIP.
I'd like having a section like this, but it isn't trivial to integrate it into the current documentation.